packages/ds-medicare-gov/src/styles/components/_Forms.scss

```diff
@@ -1,23 +1,9 @@
 @use '../layout' as *;
 
 .ds-c-field {
-  border-color: var(--text-input__border-color);
-
-  color: var(--text-input__color);
-  font-family: var(--font-family-rubik);
-  font-size: var(--font-size-md);
-  padding: $spacer-2;
```

Comment:
The update to the `text-input__padding` token came from this override - `$spacer-2` equates to `t.spacer['2']`